README.md

```diff
@@ -61,6 +61,39 @@ module "vpc" {
 For Terraform version older than 0.7.0 use `ref=v1.0.0`:
 `source = "github.com/terraform-community-modules/tf_aws_vpc?ref=v1.0.0"`
 
+Usage with other modules
+------------------------
+
+There are situations where other modules will be used alongside this module, but
+all the networking components out of this module (such as route tables, NAT
+Gateways and Internet Gateways) needs to be provisioned first before resources
+(such as EC2 instances) can be created in other modules. Use of `depends_id` as
+an output variable provides a suitable workaround.
+
+Here is an example of usage alongside the `tf_aws_bastion_s3_keys` community
```

Comment:
The real example is as per the README (and also what I'm facing at the moment). The bastion host is provisioned before the Internet Gateway in the public subnet has been provisioned. This causes userdata/bootstrapping scripts to fail, especially during apt/yum updates.